gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/FeatureSupportTest.java

```diff
@@ -587,7 +587,7 @@ public void shouldSupportRemoveEdgesIfEdgeCanBeRemoved() throws Exception {
         public void shouldSupportRemovePropertyIfAPropertyCanBeRemoved() throws Exception {
             try {
                 final Vertex v = graph.addVertex();
-                final Edge e = v.addEdge("self", v);
+                final Edge e = v.addEdge("self", v, "name", "foo");
```

Comment:
Agreed. I started with all features set to `false` then started turning things on one at a time and in different combinations.